.circleci/config.yml

```diff
@@ -124,6 +124,14 @@ commands:
       - browser-tools/install-chrome
       - browser-tools/install-chromedriver
 
+  precompile-assets:
+    description: >
+      Precompile assets
+    steps:
+      - run:
+          name: Precompile assets
+          command: bundle exec rails assets:precompile
```

Comment:
I see that in https://github.com/ministryofjustice/laa-court-data-ui/pull/1388 the `rails assets:precompile` has been replaced by `yarn install && yarn build`. Is there any benefit of one over the other?

Reply:
Our CSS and JS is compiled through Webpack so using the above command throws errors for us due to additional setup required to get the scss to compile correctly. If this works for you keep it as is 👍